app/src/pages/save-the-date.tsx

```diff
@@ -0,0 +1,341 @@
+                ...BUTTON_STYLE,
+              }}
+            >
+              {tt["translated-version"]}
+            </a>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <center>
+            <table style={CELL_GROUP_STYLE}>
+              <tbody>{clic_cells.map((cell) => StdCellComponent(cell))}</tbody>
+            </table>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <center>
+            <h1
+              style={{
+                color: save_the_date.title_color || undefined,
+                ...TITLE_STYLE,
+              }}
+            >
+              {tt["commissions"].toUpperCase()}
+            </h1>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td style={{ padding: 10 }}>
+          <table
+            style={{
+              border: "solid 1px " + save_the_date.title_color,
+              borderRadius: 25,
+              ...CELL_GROUP_STYLE,
+            }}
+          >
+            <tbody>
+              {commissions_cells.map((cell) => StdCellComponent(cell))}
+            </tbody>
+          </table>
+        </td>
+      </tr>
+
+      <tr>
+        <td
+          style={{
+            padding: "2rem",
+            color: save_the_date.text_color || undefined,
+          }}
+        >
+          <h2 style={{ color: save_the_date.title_color || undefined }}>
+            {translated_std.title}
+          </h2>
+          <Markdown>{translated_std.description}</Markdown>
+        </td>
+      </tr>
+    </>
+  );
+}
+
+function StdCellComponent(cell: SaveTheDateCell) {
+  const language_code = locale(useRouter());
+  const translation = getTranslation(cell, language_code);
+  const dateParts = (cell.date || "00-01-0000").split("-").map(Number);
+  const myDate = new Date(dateParts[0], dateParts[1] - 1, dateParts[2]);
+
+  // Use Intl.DateTimeFormat to format the date as needed
+  const formattedDate = new Intl.DateTimeFormat("en-US", {
+    weekday: "long", // Full name of the day (e.g., Monday)
+    day: "numeric", // Day of the month (e.g., 25)
+    month: "long", // Full name of the month (e.g., December)
+    year: "numeric", // Full year (e.g., 2023)
+  }).format(myDate);
+
+  return (
+    <tr key={cell.id}>
+      <td
+        key={translation.title}
+        style={{
+          color: cell.text_color || undefined,
+          background: cell.background_color || undefined,
+          ...CELL_STYLE,
+        }}
+      >
+        <table style={{ width: "100%" }}>
+          <tbody>
+            {cell.image ? (
+              <tr>
+                <td>
+                  <center>
+                    <img
+                      sizes="4rem"
+                      src={directusImageUrl(cell.image)}
+                      alt={translation.title}
+                      style={{ maxHeight: "6rem", maxWidth: "10rem" }}
+                    />
+                  </center>
```

Comment:
I think having the image left aligned (like in the emails) might look better

Reply:
Maybe aside of the title and the date is better, I'll try that :)